renderer/brave_content_settings_observer.cc

```diff
@@ -18,7 +18,10 @@
 #include "components/content_settings/core/common/content_settings_pattern.h"
 #include "components/content_settings/core/common/content_settings_utils.h"
 #include "content/public/renderer/render_frame.h"
+#include "mojo/public/cpp/bindings/remote.h"
```

Comment:
do we have a dep for this?

Reply:
No, will add.

Reply:
Fixed in 14148ce6c7e3616560ae3c67c033de93a008893c